src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/util/RcaConsts.java

```diff
@@ -53,4 +57,9 @@
   public static final long rcaConfPollerPeriodicity = 5;
   public static final long nodeRolePollerPeriodicity = 60;
   public static final TimeUnit rcaPollerPeriodicityTimeUnit = TimeUnit.SECONDS;
+
+  public static final String HOT_NODE_SUMMARY_TABLE = HotNodeSummary.class.getSimpleName();
+  public static final String HOT_CLUSTER_SUMMARY_TABLE = HotClusterSummary.class.getSimpleName();
+  public static final String HOT_RESOURCE_SUMMARY_TABLE = HotResourceSummary.class.getSimpleName();
+  public static final String HIGH_HEAP_USAGE_CLUSTER_RCA_TABLE = HighHeapUsageClusterRca.class.getSimpleName();
```

Comment:
I think it would make things a lot more clear if we move those constant strings into each Rca/Summary class itself instead of adding it here.  Maybe we can have an abstract getTableName() method in base class and derive it in each RCA/summary class ?